classes/app.rb

```diff
@@ -0,0 +1,159 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_book
+    print 'Title : '
+    title = gets.chomp
+    print 'Author : '
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    @books.push(book)
+  end
+
+  def list_books
+    if @books.empty?
+      puts "\n"
+      puts 'No books in the library'
+      return
+    end
+    @books.each do |book|
+      puts "Title : #{book.title} | Author : #{book.author}"
+    end
+    puts "\n"
+  end
+
+  # Create new person method
+  def create_person
+    puts "\n"
+    print 'Do you want to create a student (1) or a teacher (2) ? [Input a number] : '
+    puts "\n"
+    get_choice = lambda do
+      choice = gets.chomp
+      case choice
+      when '1'
+        create_student
+      when '2'
+        create_teacher
+      else
+        puts "\n"
+        puts 'Please enter a valid number [1 or 2]'
+        puts "\n"
+        get_choice.call
+      end
+      puts "\n"
+    end
+    get_choice.call
+  end
+
+  # List all people method
+  def list_people
+    if @people.empty?
+      puts "\n"
+      puts 'No person is registered in the library'
+      return
+    end
+    @people.each do |person|
+      puts "[#{person.class}] Name : #{person.name} | ID : #{person.id} | Age : #{person.age}"
+      puts "\n"
+    end
+  end
+
+  # Create new rental method
+  def create_rental
+    if @books.size.zero?
+      puts 'No books in the library'
+    elsif @people.size.zero?
+      puts 'No one registered in the library'
+    else
+      create_rental_helper
+    end
+  end
+
+  # List all rentals method
+  def list_rentals
+    puts "\n"
+    if @rentals.empty?
+      puts 'No rent is registered in the library'
+    else
+      puts 'Select a person from the following list by ID'
+      @people.each do |person|
+        puts "ID : #{person.id} => #{person.name}"
+      end
+      puts "\n"
+      print "Enter person\'s ID :"
+      person = gets.chomp
+      puts "\n"
+      @rentals.each do |rental|
+        if rental.person.id.to_i == person.to_i
+          puts "Date : #{rental.date}, Book \"#{rental.book.title}\" by : #{rental.book.author}"
+        end
+      end
+    end
+  end
+
+  private
+
+  # create new student method
+  def create_student
+    print 'Name: '
+    name = gets.chomp
+    print 'Age: '
+    age = gets.chomp
```

Comment:
If I give to student's age a wrong argument, it is not rising an error. 
You can see the image here:

<img width="529" alt="Screen Shot 2022-08-26 at 21 39 06" src="https://user-images.githubusercontent.com/69011963/186952727-0eff0bf5-eebd-440a-bf25-68dacdc9d557.png">

- [ ] Kindly fix this issue. When you give a wrong argument, make sure it shows an error!